consensus/ibft/ibft_test.go

```diff
@@ -757,6 +761,10 @@ func (t *mockTransition) Write(txn *types.Transaction) error {
 	return nil
 }
 
+func (t *mockTransition) WriteFa(txn *types.Transaction) {
```

Comment:
Please rename this method to something more descriptive

Reply:
This was leftover should have deleted it.
fixed 79a49accce1b9344e439ec8fe560eccfae0392c1